projects/addon-table/components/table/th/th.style.less

```diff
@@ -15,6 +15,8 @@
     box-sizing: border-box;
     box-shadow: 0 0.3125rem rgba(237, 237, 237, 0);
     border: 1px solid var(--tui-base-04);
+    // Create new viewport for fixed positioning
+    transform: translate3d(0, 0, 0);
```

Comment:
There's also padding reset in TD if there's a control inside.

Reply:
fixed

Reply:
Where does the editable class come from?

Reply:
fixed